### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Some-random**
Update readme with latest result of HKUST (difference being I used average_checkpoint decoding). Also trying to give README of different places different meaning

### CODE REVIEW

File: examples/asr/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ source env.sh
 bash examples/asr/$task_name/run.sh
 ```
 
-## A complete run contains following stages:
+## Stages:
```
Comment (chenguoguo) [inline examples/asr/README.md]: Let's use "Core Stages" instead.

File: examples/asr/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ source env.sh
 bash examples/asr/$task_name/run.sh
```
Comment (chenguoguo) [inline examples/asr/README.md]: I'd suggest change "Currently supported ASR tasks are HKUST and AISHELL. An example script for Librispeech will be released shortly. To perform the full procedure of ASR experiments, simply run:" to the following:

"Please refer to [the results table](#results-for-asr-tasks) for supported tasks/examples. To run an ASR example, execute the following commands from your Athena root directory:"

This way, we don't have to modify this paragraph to reflect the updated examples.

**Review by chenguoguo — COMMENTED** [review]
Minor suggestions.

File: examples/asr/README.md (hunk 2 above)
Comment (Some-random) [reply examples/asr/README.md]: Thanks for the useful comments! I've modified README accordingly.

**Review by chenguoguo — APPROVED** [review]
LGTM

**Pull request merged**